README.md

```diff
@@ -1 +0,0 @@
-# template-repo
```

Comment:
Instead of killing of your README, add a short description of what this project is about, and how to run it - something in the lines of "run yarn install and then vite". The README.md file is the first thing that pops up in the eyes of someone opening your github repository, you don't want it to be empty.